ProductivitySuite/docker/docker_build_compose.yaml

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  chatqna:
+    build:
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+        no_proxy: ${no_proxy}
+      dockerfile: ../../ChatQnA/docker/Dockerfile
+    image: ${REGISTRY:-opea}/chatqna:${TAG:-latest}
+  embedding-tei:
+    build:
+      context: GenAIComps
+      dockerfile: comps/embeddings/langchain/docker/Dockerfile
+    extends: chatqna
+    image: ${REGISTRY:-opea}/embedding-tei:${TAG:-latest}
+  retriever-redis:
+    build:
+      context: GenAIComps
+      dockerfile: comps/retrievers/langchain/redis/docker/Dockerfile
+    extends: chatqna
+    image: ${REGISTRY:-opea}/retriever-redis:${TAG:-latest}
+  reranking-tei:
+    build:
+      context: GenAIComps
+      dockerfile: comps/reranks/tei/docker/Dockerfile
+    extends: chatqna
+    image: ${REGISTRY:-opea}/reranking-tei:${TAG:-latest}
+  llm-tgi:
+    build:
+      context: GenAIComps
+      dockerfile: comps/llms/text-generation/tgi/Dockerfile
+    extends: chatqna
+    image: ${REGISTRY:-opea}/llm-tgi:${TAG:-latest}
+  dataprep-redis:
+    build:
+      context: GenAIComps
+      dockerfile: comps/dataprep/redis/langchain/docker/Dockerfile
+    extends: chatqna
+    image: ${REGISTRY:-opea}/dataprep-redis:${TAG:-latest}
+  promptregistry-mongo:
+    build:
+      context: GenAIComps
+      dockerfile: comps/prompt_registry/mongo/docker/Dockerfile
+    extends: chatqna
+    image: ${REGISTRY:-opea}/promptregistry-mongo-server:${TAG:-latest}
+  chathistory-mongo:
+    build:
+      context: GenAIComps
+      dockerfile: comps/chathistory/mongo/docker/Dockerfile
+    extends: chatqna
+    image: ${REGISTRY:-opea}/chathistory-mongo-server:${TAG:-latest}
+  productivity-suite-react-ui:
+    build:
+      context: ui
+      dockerfile: ./ui/docker/Dockerfile.react
```

Comment:
could not find /home/sdp/GenAIExamples-actions-runner/_work/GenAIExamples/GenAIExamples/ProductivitySuite/docker/ui/ui/docker: stat /home/sdp/GenAIExamples-actions-runner/_work/GenAIExamples/GenAIExamples/ProductivitySuite/docker/ui/ui/docker: no such file or directory
Error: Process completed with exit code 17.

Reply:
https://github.com/opea-project/GenAIExamples/actions/runs/10416920170/job/28850146270?pr=592#step:4:304